packages/api/controllers/nomination.js

```diff
@@ -76,10 +76,15 @@ const updateNomination = async (req, res) => {
     //can continue using additional conditional to use other email functions,
     //depending on status of application
     //current nominations don't have decline status, that should come after nominations hit ready for board review. TBD
-    if (nomination.changed('status') && nomination.status === 'Decline') {
-      sendDeclineEmail(updatedNom)
-    }
-    return res.status(200).json(nomination);
+    if (nomination.changed('status')) {
+      if (nomination.status === 'Decline') {
+        sendDeclineEmail(updatedNom)
+      }
+      if (nomination.status === 'HIPAA Verified') {
```

Comment:
It would be cool if we could write a test for this.

Reply:
Yes, updatedNom should be nomination. That should also be fixed on the main development branch as well, we have fixed it multiple times, but every time we come to a new branch it is back. That would render the email methods inoperable as it is an empty object.

Reply:
Adjusted updatedNom to nomination.

Reply:
This branch isn't up to date with the current changes from the develop branch. If you look here: https://github.com/the-difference-engine/ksf/tree/27-Log-HIPAA-Form-Timestamp you should see "6 commits ahead, 18 commits behind develop". This is why you're not seeing the latest changes from develop. If you merge develop into your branch it should be fixed.

Reply:
This branch has been merged with the develop branch and re-committed.